CHANGELOG.md

```diff
@@ -12,9 +12,13 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 - SSL support (#155)
 - IPROTO_PUSH messages support (#67)
+- Public API with request object types (#126)
 
 ### Changed
 
+- IPROTO constants that identify requests renamed from `NameRequest` to
```

Comment:
I think it's better to use something like `<Name>Request` because until I've seen the code changes I didn't understood that it meant a group of constants

Reply:
By the way, they're named `IPROTO_*` in tarantool sources.

Reply:
> I think it's better to use something like `<Name>Request` because until I've seen the code changes I didn't understood that it meant a group of constants

It confused me too. Thank you, I updated it.

Reply:
> By the way, they're named `IPROTO_*` in tarantool sources.

Thanks. I'll try to remember. Updated.